wsynphot/calibration.py

```diff
@@ -14,9 +14,7 @@ def get_vega(vega_file=None):
     if vega_file is None:
         vega_file = default_vega_path
     if not os.path.exists(vega_file):
-        raise IOError('Calibration file {0} does not exist - please download by'
```

Comment:
why not doing the download?

Reply:
@wkerzendorf I've added the download_calibration() function by using same code of commit I shared https://github.com/starkit/wsynphot/commit/3bb2c9c8ad7ef2b7b040f7577ffaa46e0541ac6f#diff-e39e153dbbd3a0ecb13c9a072d53a0a0 in PR description, with some required changes! Let me know if this is what you wanted?